### ISSUE TITLE
bypassing workers not working (master branch)

### ISSUE THREAD

**Issue by KushGoyal** [GoogleCloudPlatform/psq#26]
bypassing workers is not working as mentioned in the readme docs : https://github.com/GoogleCloudPlatform/psq#bypassing-workers-for-testing

I am getting this error:
```python
20:14:14 worker.1 | Traceback (most recent call last):
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/bin/psqworker", line 11, in <module>
20:14:14 worker.1 |     load_entry_point('psq==0.5.0', 'console_scripts', 'psqworker')()
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/click/core.py", line 664, in __call__
20:14:14 worker.1 |     return self.main(*args, **kwargs)
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/click/core.py", line 644, in main
20:14:14 worker.1 |     rv = self.invoke(ctx)
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/click/core.py", line 837, in invoke
20:14:14 worker.1 |     return ctx.invoke(self.callback, **ctx.params)
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/click/core.py", line 464, in invoke
20:14:14 worker.1 |     return callback(*args, **kwargs)
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/psq/psqworker.py", line 116, in main
20:14:14 worker.1 |     worker.listen()
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/psq/worker.py", line 94, in listen
20:14:14 worker.1 |     super(MultiprocessWorker, self).listen()
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/psq/worker.py", line 53, in listen
20:14:14 worker.1 |     tasks = self._safe_dequeue()
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/psq/worker.py", line 46, in _safe_dequeue
20:14:14 worker.1 |     return inner()
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/retrying.py", line 49, in wrapped_f
20:14:14 worker.1 |     return Retrying(*dargs, **dkw).call(f, *args, **kw)
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/retrying.py", line 212, in call
20:14:14 worker.1 |     raise attempt.get()
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/retrying.py", line 247, in get
20:14:14 worker.1 |     six.reraise(self.value[0], self.value[1], self.value[2])
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/six.py", line 686, in reraise
20:14:14 worker.1 |     raise value
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/retrying.py", line 200, in call
20:14:14 worker.1 |     attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/psq/worker.py", line 45, in inner
20:14:14 worker.1 |     return self.queue.dequeue(max=self.tasks_per_poll, block=True)
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/psq/queue.py", line 114, in dequeue
20:14:14 worker.1 |     self.subscription = self._get_or_create_subscription()
20:14:14 worker.1 |   File "/Users/kushgoyal/.virtualenvs/myproject/lib/python3.6/site-packages/psq/queue.py", line 73, in _get_or_create_subscription
20:14:14 worker.1 |     subscription_name, topic=self.topic)
20:14:14 worker.1 | AttributeError: 'Queue' object has no attribute 'topic'
```

**Comment by jonparrott** [comment]
Can I see your queue declaration?

**Comment by KushGoyal** [comment]
I am using it exactly like mentioned:
```python
q = psq.Queue(None, async=False)
r = q.enqueue(adder, 10, 15)
```

**Comment by jonparrott** [comment]
Sorry for the late response on this, but we've been working hard to improve the underlying pubsub library. Hopefully this issue is resolved with #34. But, if it's not, feel free to comment.
